docs/policy.md

```diff
@@ -202,3 +202,41 @@ policies:
 In the above example, the access to favicon is always granted and never requires an authorization.
 This feature is useful for web browsers and it is a good practice to set this policy.
 In the second policy, no-authorization access is granted to all member resources defined by a path wildcard.
+
+## Resource properties
+
+It is possible to filter allowed fields using `properties` or `blacklistProperties` properties.
+It is not possible to use both `properties` and `blacklistProperties` at the same time.
```

Comment:
If an user wrongly uses both at the same time, what will happen? Error happens on server start, or either property is preferred?

Reply:
Error happens on the server start. I've updated the docs to contain that information.

Reply:
I've also added test for that case.